evaluate_tflite.py

```diff
@@ -79,27 +83,39 @@ def main():
 
     print([x.name for x in processes])
 
+    wavlist = []
     ground_truths = []
     predictions = []
     losses = []
 
     with open(args.csv, 'r') as csvfile:
         csvreader = csv.DictReader(csvfile)
+        count = 0
         for row in csvreader:
+            count += 1
             work_todo.put({'filename': row['wav_filename'], 'transcript': row['transcript']})
+    print('Totally %d work todo\n' % count)
     work_todo.join()
+    print('\nTotally %d work done' % work_done.qsize())
 
     while (not work_done.empty()):
         msg = work_done.get()
         losses.append(0.0)
         ground_truths.append(msg['ground_truth'])
         predictions.append(msg['prediction'])
+        wavlist.append(msg['wav'])
 
     wer, cer, samples = calculate_report(ground_truths, predictions, losses)
     mean_loss = np.mean(losses)
 
     print('Test - WER: %f, CER: %f, loss: %f' %
           (wer, cer, mean_loss))
 
+    with open(args.csv + '.txt', 'w') as ftxt:
+        with open(args.csv + '.out', 'w') as fout:
```

Comment:
nit: you can comma separate context managers:

```python
with open(args.csv + '.txt', 'w') as ftxt, open(args.csv + '.out', 'w') as fout:
    # ...
```

Reply:
Thx. Will do